packages/dds/tree/src/simple-tree/core/types.ts

```diff
@@ -45,17 +63,15 @@ export type Unhydrated<T> = T;
  *
  * @sealed @public
  */
-export interface TreeChangeEvents {
+export interface TreeChangeEvents<TNode = TreeNode> {
 	/**
-	 * Emitted by a node after a batch of changes has been applied to the tree, if a change affected the node, where a
-	 * change is:
+	 * Emitted by a node after a batch of changes has been applied to the tree, if any of the changes affected the node.
 	 *
-	 * - For an object node, when the value of one of its properties changes (i.e., the property's value is set
-	 * to something else, including `undefined`).
+	 * - Object nodes define a change as the value of one of its properties changing (i.e., the property's value is set, including when set to undefined).
```

Comment:
Nit (I initially had a hard time parsing this, but understood after re-reading. Feel free to take or leave this suggestion):
```suggestion
	 * - Object nodes define a change as being when the value of one of its properties changes (i.e., the property's value is set, including when set to `undefined`).
```

Also, should it say "one or more of its properties changes"?